pages/events/index.vue

```diff
@@ -62,15 +62,18 @@
           </div>
         </template>
         <template #results>
-          <UiCard
-            v-if="noEvents"
-            :image="emptyCard.img"
-            :title="emptyCard.title"
-          >
-            <div class="event-page__empty-card-description">
-              {{ emptyCard.description }}
+          <div v-if="noEvents" class="bx--row">
```

Comment:
```suggestion
          <div v-if="noEvents" class="cds--row">
```

Since migrating to Nuxt 3, we now use a newer Carbon library; therefore,  any grid class that used to begin with `bx--`  will now use `cds--`